common/transport/mqtt/src/test/java/org/thingsboard/server/transport/mqtt/MqttTransportHandlerTest.java

```diff
@@ -110,6 +111,13 @@ MqttPublishMessage getMqttPublishMessage() {
         return new MqttPublishMessage(mqttFixedHeader, variableHeader, payload);
     }
 
+    MqttPublishMessage getMqttPublishMessageAndProvisionResponse() {
```

Comment:
It may be better to rename the method to getMqttSubscribeMessageForProvisionResponseTopic